src/components/link.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+export const Link = (
+  props: React.DetailedHTMLProps<
+    React.AnchorHTMLAttributes<HTMLAnchorElement>,
+    HTMLAnchorElement
+  >
+) => {
+  const { children, style, ...restProps } = props;
+  return (
+    <a
+      target="_blank"
+      rel="noreferrer"
```

Comment:
How about `noopener` also (in fact we might want the referrer in some case I suppose)
```suggestion
      rel="noreferrer noopener"
```